dashboard/app/models/user.rb

```diff
@@ -1506,6 +1506,10 @@ def under_13?
     age.nil? || age.to_i < 13
   end
 
+  def over_21?
```

Comment:
This is slightly different from the under_13 logic because we want to assume a user is under 13 if we don't have their age. Here, we only want to classify a user as over 21 if we have their age, and it's 21+